cityofaustin/atd-moped
# Fix bug where the aerial basemap renders on top of all other layers

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13097

This PR sorts out the order that basemap, component, and street label layers render. ~These updates introduce some Mapbox errors in the console that I'm continuing to look at, but I wanted to get this code out to review.~ _Update: The errors are fixed now._

## Testing
**URL to test:** 

Local

**Steps to test:**
Testing this is tricky since deploy previews are not on the approved list for using the Nearmap token for the aerial imagery. It is best to look at this one locally.

1. Test locally and go to a project with components and related components (or add components and a subproject with components to an existing project)
2. Go to the project summary page and use the basemap control to show the **Aerial** layer. You should:
   - see the basemap update to aerial imagery
   - see the project's component features in blue (or related components in green) on top of the the aerial basemap
   - see the street labels on top of both the aerial basemap and the component and related features
3. Go to the project Map tab and repeat the same steps. You can also try adding new components or editing existing ones.

---
#### Ship list
